ts/features/wallet/component/features/PaymentMethodInitiatives.tsx

```diff
@@ -86,6 +96,19 @@ const PaymentMethodInitiatives = (props: Props): React.ReactElement | null => {
     props.paymentMethod,
     props.bpdRemoteConfig
   );
+  const { isIdPayEnabled, initiativeListPot } = props;
+  const navigation = useNavigation<IOStackNavigationProp<WalletParamsList>>();
+  const namedInitiativesList = pot
+    .getOrElse(initiativeListPot, [])
+    .filter(initiative => initiative.initiativeName !== undefined);
```

Comment:
Why is initiativeName not required?

Reply:
`WalletDTO` is, at the end of the day, its type and it marks `initiativeName` as optional so possibly undefined.
Realistically, I have no idea in which case an initiative might have no name, other than maybe some storing error.